modules/feature-upload.php

```diff
@@ -291,7 +304,7 @@
 				exit();
 			}
 			
-			$filepath = $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
+			$filepath = getcwd() . "/" . $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
```

Comment:
Sorry this took so long to get back to -- work has been nuts this past week.

I tested this issue on the two other environments I have available -- a linux server and another Windows machine, and could not reproduce the issue with either of them.  I can't seem to track down the issue, but passing it to imagick as an absolute path solves the problem for me.  I went ahead an made a commit wrapping the path in a realpath() call.

Thank you for being so open to contributions! =D  I've enjoyed working with this codebase.